template.rb

```diff
@@ -106,4 +114,8 @@ def print_security_update_message
   print_wrapped set_color(message.gsub("\n", ' '), :yellow)
 end
 
+def quick_update?
+  ENV.fetch('QUICK_UPDATE', nil)
+end
```

Comment:
I added a `QUICK_UPDATE` config to the template which, when used with the `bin/quick_update_script`, makes it easy to quickly update the sandbox during development.

Reply:
@kennyadsl Okay, I removed the commit for the quick update feature. Will submit a separate PR for it :+1: